mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -196,8 +196,14 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_24833 getBoundingBox (Lnet/minecraft/class_4050;)Lnet/minecraft/class_238;
 		ARG 1 pose
 	METHOD method_25937 displaySoulSpeedEffects ()V
-	METHOD method_26082 applyStatusEffect (Lnet/minecraft/class_1293;Lnet/minecraft/class_1297;)V
+	METHOD method_26082 setStatusEffect (Lnet/minecraft/class_1293;Lnet/minecraft/class_1297;)V
+		COMMENT Sets a status effect in this entity.
```

Comment:
I would document that this is for client-side effects for players. The name could possibly be more clear too.

Reply:
Agreed, though I will put this in an `@apiNote` since when called on the server, this is effective too.